client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -46,8 +43,7 @@ const Gutenboard: React.FunctionComponent = () => {
 	const { showSignupDialog, onSignupDialogClose } = useSignup();
 	const effectiveFontPairings = useFontPairings();
 
-	const isA8cTeamMember = useSelect( ( select ) => select( READER_STORE ).isA8cTeamMember() );
-	useBodyClass( 'font-smoothing-antialiased', isA8cTeamMember );
+	window.document.body.classList.add( 'font-smoothing-antialiased' );
```

Comment:
I don't think it's good practice to run side-effects directly inside the body of the component. `useBodyClass()` is still good for this case. But if you think it's overkill (there's nothing async going on so maybe it is), then you can just wrap in a `useEffect()` with `[]` for the dependency list.

Reply:
Going with `useEffect()` with `[]`. https://github.com/Automattic/wp-calypso/pull/54196/commits/8206296cd90e41e54362029127d5bee4cc21a201